syz-manager/manager.go

```diff
@@ -1081,6 +1081,17 @@ func (mgr *Manager) fuzzerConnect(modules []*host.KernelModule) (
 			log.Fatalf("failed to create coverage filter: %v", err)
 		}
 		mgr.modulesInitialized = true
+	} else {
+		for _, m1 := range modules {
```

Comment:
This is a useful check, but I think we want a stricter check: we want exact equality of these lists. Say, if there any new modules loaded, (1) they are not present in mgr.modules, (2) they can be loaded at different address by new VMs and this check won't catch it.
Sort them if they are not sorted yet and compare for exact equality. Maybe reflect.DeepEqual will work.
And move this to a separate commit. This looks useful on its own and unrelated to the rest of the changes in this commit.

Reply:
Let me think and put into another PR.